src/btree/bt_curprev.c

```diff
@@ -678,7 +689,7 @@ __wt_btcur_prev_prefix(WT_CURSOR_BTREE *cbt, WT_ITEM *prefix, bool truncating)
             if (ret != WT_NOTFOUND)
                 break;
         }
-
+skip_page:
```

Comment:
As in the case of next, we should skip checking for delete count if we did not even read the page.

Reply:
Fixed.